deploy-as-code/helm/charts/digit-works/backend/works-inbox-service/values.yaml

```diff
@@ -0,0 +1,73 @@
+# Common Labels
+labels:
+  app: "works-inbox-service"
+  group: "digit-works"
+
+namespace: works
+
+# Ingress Configs
+ingress:
+  namespace: egov
+  enabled: true
+  zuul: true
+  context: "works-inbox-service"
+
+# Container Configs
+image:
+  repository: "works-inbox-service"
+replicas: "1"
+healthChecks:
+  enabled: true
+  livenessProbePath: "/works-inbox-service/health"
+  readinessProbePath: "/works-inbox-service/health"
+appType: "java-spring"
+tracing-enabled: true
+heap: "-Xmx192m -Xms192m"
+java-args: ""
+
+# Additional Container Envs
+env: |
+  - name: SERVER_PORT
+    value: "8080"
+  - name: JAVA_OPTS
+    value: {{ index .Values "heap" | quote }}
+  - name: JAVA_ARGS
+    value: {{ index .Values "java-args" | quote }}
+  {{- if index .Values "tracing-enabled" }}
+  - name: TRACER_OPENTRACING_ENABLED
+    value: "true" 
+  {{- end }}
+  - name: SERVER_CONTEXT_PATH
+    value: /works-inbox-service
+  - name: SERVER_SERVLET_CONTEXT-PATH
+    value: /works-inbox-service 
+  - name: EGOV_MDMS_HOST
+    valueFrom:
+      configMapKeyRef:
+        name: egov-service-host
+        key: egov-mdms-service
+  {{- if index .Values "java-enable-debug" }}
+  - name: JAVA_ENABLE_DEBUG
+    value: "true"
+  {{- end }}
+  - name: EGOV_WORKFLOW_HOST
+    valueFrom:
+      configMapKeyRef:
+        name: egov-service-host
+        key: egov-workflow-v2
+  {{- if index .Values "es-search-pagination-default-limit" }}
+  - name: ES_SEARCH_PAGINATION_DEFAULT_LIMIT
+    value: {{ index .Values "es-search-pagination-default-limit" | quote }} 
+  {{- end }}
+  {{- if index .Values "es-search-pagination-max-search-limit" }}
+  - name: ES_SEARCH_PAGINATION_MAX_SEARCH_LIMIT
+    value: {{ index .Values "es-search-pagination-max-search-limit" | quote }} 
+  {{- end }}
+  {{- if index .Values "es-search-default-sort-order" }}
+  - name: ES_SEARCH_DEFAULT_SORT_ORDER
+    value: {{ index .Values "es-search-default-sort-order" | quote }} 
+  {{- end }}
+  {{- if index .Values "state-level-tenant-id" }}
+  - name: STATE_LEVEL_TENANT_ID
+    value: {{ index .Values "state-level-tenant-id" | quote }} 
```

Comment:
Don't read it from there. Either remove it of not required, or read it from the config map.